Modules/Messages/src/Messages.Application/Messages/Commands/SendMessage/Validator.cs

```diff
@@ -21,7 +22,9 @@ public Validator()
                 .SetValidator(new SendMessageCommandRecipientInformationValidator()));
 
         RuleFor(m => m.Recipients.Count)
-            .InclusiveBetween(1, 50).WithErrorCode(GenericApplicationErrors.Validation.InvalidPropertyValue().Code);
+            .LessThanOrEqualTo(options.Value.MaxNumberOfMessageRecipients)
+            .WithName("Recipients")
```

Comment:
No need for a custom name, if I am not mistaken.

I guess you copied it from above. But there we only add it because due to the fact that it's an element of a list, the error message would be weird otherwise.